Userland/Libraries/LibCore/System.cpp

```diff
@@ -116,6 +116,18 @@ ErrorOr<void> mount(int source_fd, StringView target, StringView fs_type, int fl
     HANDLE_SYSCALL_RETURN_VALUE("mount", rc, {});
 }
 
+ErrorOr<void> umount(StringView mount_point)
+{
+    if (mount_point.is_null())
+        return Error::from_errno(EFAULT);
+
+    Syscall::SC_umount_params params {
+        { mount_point.characters_without_null_termination(), mount_point.length() },
+    };
+    int rc = syscall(SC_umount, &params);
```

Comment:
huh? this syscall doesn't accept `SC_umount_params` it accepts a pointer and length, this won't work.

Reply:
Thank you, now I understood better how syscalls works :)
Fixed